.env

```diff
@@ -0,0 +1,5 @@
+GITHUB_API_TOKEN: 1234
```

Comment:
Shouldn't we also add other keys like `MAILCHIMP_API_KEY`, `MAILCHIMP_LIST_ID` etc? :thinking:

Reply:
Ahh, good idea.